server/ai/anthropic/client.go

```diff
@@ -203,3 +199,9 @@ func (c *Client) MessageCompletion(completionRequest MessageRequest) (*ai.TextSt
 
 	return &ai.TextStreamResult{Stream: output, Err: errChan}, nil
 }
+
+func (c *Client) prepareRequestHeaders(req *http.Request) {
```

Comment:
Not sure this helper is necessary. But if we do want to keep it maybe call it `setAnthropicHeaders` and pull in the streaming ones with a boolean parameter.

Reply:
That sounds good. Yeah it was repeated logic so I figured not repeating it would be better, especially because then there were less hard-coded strings and consolidating into one place would eliminate the argument for making constants to reduce the number of hard-coded strings.

Reply:
I just remembered my reasoning for originally calling it `prepareRequestHeaders()`.
using "set" typically denoted that you are setting an instance variable on the calling object, whereas in this case I am setting values of something being passed in.